tests/services/supabase/test_users_manager.py

```diff
@@ -14,6 +14,7 @@
     INSTALLATION_ID,
     UNIQUE_ISSUE_ID,
     NEW_INSTALLATION_ID,
+    EMAIL,
```

Comment:
I think EMAIL should cover all patterns we discussed?

Reply:
I'm not sure what you mean. you want me to create and test multiple emails?